client/components/site-selector/add-site.jsx

```diff
@@ -10,23 +10,90 @@ import Gridicon from 'gridicons';
 /**
  * Internal dependencies
  */
+import config from 'config';
 import Button from 'components/button';
+import PopoverMenu from 'components/popover/menu';
+import PopoverMenuItem from 'components/popover/menu-item';
 import { recordTracksEvent } from 'state/analytics/actions';
 
 class SiteSelectorAddSite extends Component {
-	getAddNewSiteUrl() {
-		return '/jetpack/new/?ref=calypso-selector';
+	state = {
+		showPopoverMenu: false,
+	};
+
+	handleShowPopover = isShowing => {
+		const action = isShowing ? 'show' : 'hide';
+
+		this.setState( {
+			showPopoverMenu: isShowing,
+		} );
+
+		this.props.recordTracksEvent( 'calypso_add_site_popover', { action } );
+	};
+
+	onPopoverButtonClick = () => {
+		this.handleShowPopover( ! this.state.showPopoverMenu );
+	};
+
+	onClosePopover = () => {
+		this.handleShowPopover( false );
+	};
+
+	getNewWpcomSiteUrl() {
+		return config( 'signup_url' ) + '?ref=calypso-selector';
+	}
+
+	getNewJetpackSiteUrl() {
+		return '/jetpack/connect/?ref=calypso-selector';
 	}
 
-	recordAddNewSite = () => {
-		this.props.recordTracksEvent( 'calypso_add_new_wordpress_click' );
+	recordPopoverAddNewSite = () => {
+		this.props.recordTracksEvent( 'calypso_add_new_wordpress_popover_click' );
+	};
+
+	recordPopoverAddJetpackSite = () => {
+		this.props.recordTracksEvent( 'calypso_add_jetpack_site_popover_click' );
+	};
+
+	popoverToggleRef = node => {
+		this.popoverToggle = node;
 	};
 
 	render() {
 		const { translate } = this.props;
 		return (
 			<span className="site-selector__add-new-site">
-				<Button borderless href={ this.getAddNewSiteUrl() } onClick={ this.recordAddNewSite }>
+				<PopoverMenu
+					className="site-selector__popover"
+					isVisible={ this.state.showPopoverMenu }
+					onClose={ this.onClosePopover }
+					context={ this.popoverToggle }
+				>
+					<PopoverMenuItem
+						href={ this.getNewWpcomSiteUrl() }
+						onClick={ this.recordPopoverAddNewSite }
+					>
+						<span>
+							<span>{ translate( 'Create a new site' ) }</span>
+							<span className="site-selector__popover-item-details">
+								{ translate( 'Start a new WordPress.com site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Start a new site on WordPress.com' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Thanks, bot! 🤖